modules/ossm-about-deployment-and-update-strategies.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: Concept
+[id="ossm-about-deployment-and-update-strategies"]
+= About update strategies
+:context: ossm-about-deployment-and-update-strategies
+
+The update strategy affects how the update process is performed. For each mesh, you select one of two strategies:
+
+* `InPlace`
+* `RevisionBased`
+
+The default strategy is the `InPlace` strategy.
+
+For more information, see the following documentation:
+
+* "About InPlace strategy"
+* "About RevisionBased strategy"
```

Comment:
I think this should be moved to this section https://75532--ocpdocs-pr.netlify.app/openshift-service-mesh/latest/install/ossm-installing-openshift-service-mesh.html#ossm-about-deployment-and-update-strategies

Reply:
created OSSM-6632 to follow up on this